### ISSUE TITLE
browser level check unreliable? sleipnir, k-meleon, greenbrowser, slimbrowser, deepnet

### ISSUE THREAD

**Issue by lightbeard** [ded/bowser#73]
[sleipnir](http://www.fenrir-inc.com/us/sleipnir/)
Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36 Sleipnir/6.1.4
```javascript
bowser.chrome     // true
bowser.version    // 39
bowser.a          // true
```
[k-meleon](http://kmeleon.sourceforge.net/)
Mozilla/5.0 (Windows NT 6.1; WOW64; rv:24.0) Gecko/20140105 Firefox/24.0 K-Meleon/74.0
```javascript
bowser.firefox    // true
bowser.version    // 24
bowser.a          // true
```
[greenbrowser](http://www.morequick.com/indexen.htm)
Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.1; WOW64; Trident/7.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; Media Center PC 6.0; .NET4.0C; .NET4.0E; InfoPath.2; GreenBrowser)
```javascript
bowser.msie       // true
bowser.version    // 7
bowser.c          // true
```
[deepnet](http://www.deepnetexplorer.com)
Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.1; WOW64; Trident/7.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; Media Center PC 6.0; .NET4.0C; .NET4.0E; InfoPath.2)
```javascript
bowser.msie       // true
bowser.version    // 7
bowser.c          // true
```
[slimbrowser](http://www.slimbrowser.net/en/)
Mozilla/5.0 (Windows NT 6.1; WOW64; Trident/7.0; rv:11.0) like Gecko
```javascript
bowser.msie       // true
bowser.version    // 11
bowser.a          // true
```
Is bowser working as designed?

Basically we don't see how the browser level checks such as `bowser.msie` and `boswer.firefox` can be trusted at all.

**Comment by lancedikson** [comment]
@lightbeard, hey, good catch, thanks. We will add support for all of these browsers. 

And yes, bowser is working as designed. We need support browser detection process manually. In most cases bowsers' assumptions are true, but sometimes there are special cases like those in the issue.
